assets/css/tablesorter.theme.glotpress.css

```diff
@@ -19,6 +19,7 @@ GlotPress Theme
 	cursor: pointer;
 	text-align: center;
 	border: 1px solid #eee;
+	padding-right: 20px !important;
```

Comment:
@toolstack If you use the correct selectors (see my example) you should get the padding only when it's really needed.